src/index.css

```diff
@@ -152,3 +152,43 @@ code {
   font-family: 'Montserrat', sans-serif;
   font-style: normal;
 }
+
+.bn-onboard-modal {
+  background: rgba(0, 0, 0, 0.6) !important;
+  z-index: 10000 !important;
```

Comment:
Do we need the important? and do we need the huge z-index value?

Reply:
I currently have each layer's z-index separated by factors of 10, so I've just done the same thing here to put the backdrop above the navigation. Think I should reduce z-index values all around? 

And it seems that `onboardjs`'s backdrop is for some reason the only thing that can't be modified without adding `!important` to the `css`. Haven't yet found any other way around it.

Reply:
Yeah, I had forgotten we were using an external lib... It's aight then, that's usually what follows with these kinds of things haha. And nope, the z-index is fine! Just wanted to make sure it wasn't some "nuclear" value.